src/declaration.h

```diff
@@ -580,10 +580,10 @@ class FuncDeclaration : public Declaration
                                         // scopes from having the same name
     VarDeclaration *vthis;              // 'this' parameter (member and nested)
     VarDeclaration *v_arguments;        // '_arguments' parameter
-#ifdef IN_GCC
+#ifdef IN_GCC // %%
```

Comment:
Any particular reason for this change? This same change made it's way in in multiple other places as well.